### PR TITLE
added node.js 1.7.0 rns

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ Adding Node.js Release Notes. 
* Jira/GitHub Issue numbers (if any): \ FFM-11214
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes: version released to at least one prod environment.

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
 
+### Node.js SDK 
+
+#### Version 1.7.0
+
+ - Metrics: refactor into class. (FFM-11022)
+ - Applies standard Metrics Enhancements. (FFM-11022)
+ - Fixed issue where only the first prerequisite was being processed. (FFM-9291)
+ - Add `Harness-SDK-Info`, `Harness-AccountID`, or `Harness-EnvironmentID` headers. (FFM-7051)
+ - Add support for custom TLS CAs. (FFM-7017)
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:65]:
```suggestion
 - Add support for custom TLS CA certificates. (FFM-7017)
```

**Review by douglas-j-bothwell — APPROVED** [review]
Additional feedback in Slack

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -54,6 +54,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
  -- increasing the capacity of dictionary, and
  -- additional cache recovery checks. (FFM-11089)
 
+### Node.js SDK 
+
+#### Version 1.7.0
+
+ - Node SDK has metrics hardening applied. (FFM-11022)
```
Comment (SmritiSatya) [inline release-notes/feature-flags.md:61]: Applies metrics hardening to Node SDK.

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -54,6 +54,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+ - Applies standard Metrics Enhancements. (FFM-11022)
+ - Fixed issue where only the first prerequisite was being processed. (FFM-9291)
+ - Add `Harness-SDK-Info`, `Harness-AccountID`, or `Harness-EnvironmentID` headers. (FFM-7051)
+ - Add support for custom TLS CA certificates. (FFM-7017)
```
Comment (SmritiSatya) [inline release-notes/feature-flags.md:65]: Adds support for ...

File: release-notes/feature-flags.md [hunk 4]
```diff
@@ -54,6 +54,16 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
  -- additional cache recovery checks. (FFM-11089)
 
+### Node.js SDK 
+
+#### Version 1.7.0
+
+ - Node SDK has metrics hardening applied. (FFM-11022)
+ - Applies standard Metrics Enhancements. (FFM-11022)
+ - Fixed issue where only the first prerequisite was being processed. (FFM-9291)
+ - Add `Harness-SDK-Info`, `Harness-AccountID`, or `Harness-EnvironmentID` headers. (FFM-7051)
```
Comment (SmritiSatya) [inline release-notes/feature-flags.md:64]: Adds ...

**Review by SmritiSatya — DISMISSED** [review]

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**